modules/BundledPackages/BundledPackages.proj

```diff
@@ -32,16 +39,21 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="MicroBuild.Core" Version="$(MicroBuildCorePackageVersion)" />
     <PackageReference Include="Internal.AspNetCore.Sdk" Version="$(Version)" />
   </ItemGroup>
 
+  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
+  <Target Name="noop" />
```

Comment:
Is this here also just for the MSBuild workaround? Maybe add the same comment here as elsewhere with the noop?

Reply:
Yes, it's a workaround. I'll add a comment explaining why it's here.